scripts/release

```diff
@@ -407,6 +407,23 @@ publish_release() {
   fi
 }
 
+publish_podspec() {
+  cd "$rootdir"
+  echo "Running a pod spec lint to validate the podspec..."
+  # Checking spec lint
+  pod spec lint MaterialComponents.podspec --allow-warnings > lint_logs.log
```

Comment:
I'm pretty sure `pod trunk push` does a full `pod lint` before proceeding, actually, so you can probably remove the pre-lint

Reply:
You are correct! I was trying to follow the guidelines as seen here: https://guides.cocoapods.org/making/specs-and-specs-repo.html#how-do-i-update-an-existing-pod

Thought maybe the trunk push fails slower because it has prior steps to conduct but doesn't look like it: https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/command/repo/push.rb#L73

I'll remove the linting